internal/provider/cluster_test.go

```diff
@@ -589,7 +589,7 @@ func TestUnitPopulateClusterDataDesc(t *testing.T) {
 			"tags":               types.ListNull(types.MapType{ElemType: types.StringType}),
 			"headNode":           types.MapNull(types.StringType),
 			"failures":           types.ListNull(types.MapType{ElemType: types.StringType}),
-			"loginNodes":         types.ObjectNull(loginNodesObjectTypes),
+			"loginNodes":         types.ListNull(types.ObjectType{AttrTypes: loginNodesObjectTypes}),
```

Comment:
This test must be extended to cover the new loginnodes structure. 
In practice, we should include the login nodes in the Case01 object in `contents`.

Reply:
thanks for adding the test